docs/src/ScrollableContainer.doc.js

```diff
@@ -624,25 +624,20 @@ function ScrollableContainerExample() {
 );
 
 card(
-  <MainSection
-    name="Related"
-    description={`
-
-[Box](/Box)
-
-Box's overflow prop specifies what should happen if the content is larger than the bounding box. Box should not be replaced with ScrollableContainer if the goal is simply to allow the Box to scroll when content overflows. ScrollableContainer is only needed when anchored components, such as [Tooltip](/Tooltip), [Flyout](/Flyout), [Typeahead](/Typeahead) and [Dropdown](/Dropdown), are used within a container that could potentially scroll.
-
-
-[Modal](/Modal) / [Sheet](/Sheet)
-
-Modal and Sheet come with ScrollableContainer built-in, so any anchored components used in their children tree should work out-of-the-box. Passing an additional ScrollableContainer will break the existing styling on scroll.
-
+  <MainSection name="Related">
+    <MainSection.Subsection
+      description={`
+      **[Box](/Box)**
+      Box's [overflow prop](/Box#Overflow) specifies what should happen if the content is larger than the bounding box. Box should not be replaced with ScrollableContainer if the goal is simply to allow the Box to scroll when content overflows. ScrollableContainer is only needed when anchored components, such as [Tooltip](/Tooltip), [Flyout](/Flyout), [Typeahead](/Typeahead) and [Dropdown](/Dropdown), are used within a container that could potentially scroll.
 
-[Tooltip](/Tooltip) / [Flyout](/Flyout) / [Typeahead](/Typeahead) / [Dropdown](/Dropdown)
+      **[Modal](/Modal)** / **[Sheet](/Sheet)**
+      Modal and Sheet come with ScrollableContainer built-in, so any anchored components used in their children tree should work out-of-the-box. Passing an additional ScrollableContainer will break the existing styling on scroll.
 
-ScrollableContainer must be used around any of these components if they are used within a container that could possibly scroll. This is necessary to ensure the component remains attached to its anchor on scroll. If they are located within scrolling Modal and Sheet components, ScrollableContainer isn't needed as ScrollableContainer is already built-in.
+      **[Tooltip](/Tooltip)** / **[Flyout](/Flyout)** / **[Typeahead](/Typeahead)** / **[Dropdown](/Dropdown)**
+      ScrollableContainer must be used around any of these components if they are used within a container that could possibly scroll. This is necessary to ensure the component remains attached to its anchor on scroll. If they are located within scrolling Modal and Sheet components, ScrollableContainer isn't needed as ScrollableContainer is already built-in.
```

Comment:
s/and/or in "scrolling Modal and Sheet components"

The last sentence repeats "ScrollableContainer" a lot…do you think it's clear enough to say "ScrollableContainer isn't needed as it is already built-in"?